nanoweb.py

```diff
@@ -165,6 +165,7 @@ async def handle(self, reader, writer):
                             break
                     else:
                         # 3. Try to load index file
+                        await write(request, "HTTP/1.1 200 OK\r\n\r\n")
```

Comment:
`send_file` can raise HttpError and in this case status 404 would be sent after status 200

Reply:
In line 95-97:
```python
            if isinstance(handler, str):
                await write(request, "HTTP/1.1 200 OK\r\n\r\n")
                await send_file(request, handler)
```
That might also cause this situation. Maybe this should also be changed?